ui/aboutPC.html

```diff
@@ -1,23 +1,5982 @@
+                    margin-left: 180px;
+                }
+            }
+            abbr[title],
+            abbr[data-original-title] {
+                cursor: help;
+                border-bottom: 1px dotted #999;
+            }
+            .initialism {
+                font-size: 90%;
+                text-transform: uppercase;
+            }
+            blockquote {
+                padding: 10px 20px;
+                margin: 0 0 20px;
+                font-size: 17.5px;
+                border-left: 5px solid #eee;
+            }
+            blockquote p:last-child,
+            blockquote ul:last-child,
+            blockquote ol:last-child {
+                margin-bottom: 0;
+            }
+            blockquote footer,
+            blockquote small,
+            blockquote .small {
+                display: block;
+                font-size: 80%;
+                line-height: 1.42857143;
+                color: #999;
+            }
+            blockquote footer:before,
+            blockquote small:before,
+            blockquote .small:before {
+                content: '\2014 \00A0';
+            }
+            .blockquote-reverse,
+            blockquote.pull-right {
+                padding-right: 15px;
+                padding-left: 0;
+                text-align: right;
+                border-right: 5px solid #eee;
+                border-left: 0;
+            }
+            .blockquote-reverse footer:before,
+            blockquote.pull-right footer:before,
+            .blockquote-reverse small:before,
+            blockquote.pull-right small:before,
+            .blockquote-reverse .small:before,
+            blockquote.pull-right .small:before {
+                content: '';
+            }
+            .blockquote-reverse footer:after,
+            blockquote.pull-right footer:after,
+            .blockquote-reverse small:after,
+            blockquote.pull-right small:after,
+            .blockquote-reverse .small:after,
+            blockquote.pull-right .small:after {
+                content: '\00A0 \2014';
+            }
+            blockquote:before,
+            blockquote:after {
+                content: "";
+            }
+            address {
+                margin-bottom: 20px;
+                font-style: normal;
+                line-height: 1.42857143;
+            }
+            code,
+            kbd,
+            pre,
+            samp {
+                font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+            }
+            code {
+                padding: 2px 4px;
+                font-size: 90%;
+                color: #c7254e;
+                white-space: nowrap;
+                background-color: #f9f2f4;
+                border-radius: 4px;
+            }
+            kbd {
+                padding: 2px 4px;
+                font-size: 90%;
+                color: #fff;
+                background-color: #333;
+                border-radius: 3px;

```

Comment:
Since very few of these css elements have been used in the About Me page, a good idea would be to just declare those elements in <style>. 

The other option is to include just a reference to the external bootstrap style sheet.